File: Trabalhos Teoricos/TrabalhoTeorico2/02 - Matriz em C-C++/Matriz.c
#include <stdio.h>

int main() {
    //Declarando variaveis
    int tamanho;
    int matriz[10][10];
    int soma[10];

    //Lendo tamanho
    scanf("%d", &tamanho);

    //Lendo valores
    for (int i = 0; i < tamanho; i = i + 1) {
        for (int j = 0; j < tamanho; j = j + 1) {
            scanf("%d", &matriz[i][j]);
        }   
    }

    //Somando valores
    for (int m = 0; m < tamanho; m = m + 1) {
        soma[m] = 0;
        for (int n = 0; n < tamanho; n = n + 1) {
            soma[m] = soma[m] + matriz[n][m];
        }
    }

    //Mostrar resultado
    for (int p = 0; p < tamanho; p = p + 1) {
        printf("%d ", soma[p]);
    }

    return 0;
}

File: Trabalhos Teoricos/TrabalhoTeorico2/01 - Matriz em Java/Matriz.java
public class Matriz {
    public static void main(String[] args) {
        //Declarando as variaveis
        int tamanho = 0;
        int matriz[][] = new int[10][10];
        int soma[] = new int[10];

        //Lendo o tamanho
        tamanho = MyIO.readInt();

        //Ler os elementos e adicionar na matriz
        for (int i = 0; i < tamanho; i = i + 1) {
            for (int j = 0; j < tamanho; j = j + 1) {
                matriz[i][j]= MyIO.readInt();
            }
        }

        //Somar cada coluna
        for (int k = 0; k < tamanho; k = k + 1) {
            soma[k] = 0;
            for (int l = 0; l < tamanho; l = l + 1) {
                soma[k] = soma[k] + matriz[l][k];
            }
        }
        
        //Saida
        for (int m = 0; m < tamanho; m = m + 1) {
            MyIO.print(soma[m] + " ");
        }
    }
}
